File: New folder/notes.cpp
// cpp notes
#include <iostream>
#include <vector>

std::string greet(std::string name) {
    return "hello " + name;
}

int main() {
    std::cout << greet("practice") << std::endl;

    std::vector<int> nums = {1, 2, 3};
    for (int n : nums) {
        std::cout << n * 2 << std::endl;
    }
    return 0;
}
// TODO: try templates next

File: New folder/notes.java
// java notes
public class Notes {
    static String greet(String name) {
        return "hello " + name;
    }

    public static void main(String[] args) {
        System.out.println(greet("practice"));

        int[] nums = {1, 2, 3};
        for (int n : nums) {
            System.out.println(n * 2);
        }
    }
}
// TODO: try interfaces next
